pom.xml

```diff
@@ -147,6 +147,19 @@ limitations under the License.
         <log4j-silent.properties>file:target/test-classes/log4j-silent.properties</log4j-silent.properties>
 
         <muteTestLogs>false</muteTestLogs>
+
+        <!--
+        Configuration properties for the OSGi maven-bundle-plugin
+        -->
+        <osgi.embed></osgi.embed>
```

Comment:
again, not sure i'm seeing the need for properties here - any reason you don't just add this stuff directly to `<instructions>` of the `<maven-bundle-plugin>`